src/components/divider/divider.d.ts

```diff
@@ -0,0 +1,19 @@
+import * as React from 'react';
+import { FormFieldSpacing } from '../../utils/helpers/options-helper/options-helper';
+
+export interface DividerProps {
+  /** Margin top, this value will be multiplied by the theme spacing constant (8) */
+  marginTop: FormFieldSpacing;
+
```

Comment:
do we need white lines here ?

Reply:
All the other components seem to so I just followed that convention, happy to remove though